jsonld/jsonld.go

```diff
@@ -0,0 +1,35 @@
+package jsonld
+
+import "github.com/nuts-foundation/nuts-node/core"
+
+type jsonld struct {
+	config         Config
+	contextManager ContextManager
+}
+
+// NewJSONLDInstance creates a new instance of the jsonld struct which implements the JSONLD interface
+func NewJSONLDInstance() JSONLD {
```

Comment:
all methods on JSONLD should have been placed on ContextManager

Reply:
I merged the logic of the ContextManager into the JSONLD engine.